File: csharp/DVSClient/DVSClient/exceptions/NotFoundException.cs
namespace DVSClient.Exceptions
{
    /// <summary>
    /// Represents an exception that is thrown when a requested resource is not found in the DVSClient.
    /// </summary>
    public class NotFoundException : EDVSException
    {
        /// <summary>
        /// Initializes a new instance of the <see cref="NotFoundException"/> class with a specified error message.
        /// </summary>
        /// <param name="message">The error message that explains the reason for the exception.</param>
        public NotFoundException(string? message) : base(message)
        {
        }
    }
}

File: java/src/main/java/com/experian/dvs/client/exceptions/NotFoundException.java
package com.experian.dvs.client.exceptions;

/**
 * Represents an exception that is thrown when a requested resource is not found in the DVSClient.
 */
public class NotFoundException extends EDVSException {

    /**
     * Initializes a new instance of the {@link NotFoundException} class with a specified error message.
     *
     * @param message The error message that explains the reason for the exception.
     */
    public NotFoundException(String message) {
        super(message);
    }
}


